### PR TITLE
When accounting for direct implementors, skip over private ones

### LINKED ISSUE #1623: Uint8List inherits from TypedData but docs don't show it
`Uint8List`: https://master-docs-flutter-io.firebaseapp.com/flutter/dart-typed_data/Uint8List-class.html
...has a `buffer` property, which links to `TypedData`'s `buffer`: https://master-docs-flutter-io.firebaseapp.com/flutter/dart-typed_data/TypedData/buffer.html
...which implies that `Uint8List` is related to `TypedData`, but this fact appears neither on the docs for `Uint8List` nor those from `TypedData`: https://master-docs-flutter-io.firebaseapp.com/flutter/dart-typed_data/TypedData-class.html

### LINKED ISSUE #2094: Inheritance doesn't match Implementors for ErrorDescription/DiagnosticsProperty (probably due to generics?)
https://api.flutter.dev/flutter/foundation/ErrorDescription-class.html says it inherits from "DiagnosticsProperty", but https://api.flutter.dev/flutter/foundation/DiagnosticsProperty-class.html does not list "ErrorDescription" in its list of implementors. The lack of symmetry is confusing.

### PR DESCRIPTION
**Pull request by srawlins**
Fixes #1623

### CODE REVIEW

File: lib/src/model/package_graph.dart [hunk 1]
```diff
@@ -202,7 +202,7 @@ class PackageGraph {
       allInheritableElements = {};
 
   /// Map of Class.href to a list of classes implementing that class
```
Comment (jcollins-g) [inline lib/src/model/package_graph.dart:204]: comment is now out of date

**Review by jcollins-g — APPROVED** [review]
Nice!

File: lib/src/model/package_graph.dart (hunk 1 above)
Comment (srawlins) [reply lib/src/model/package_graph.dart]: Good catch! Done.

**Comment by Hixie** [comment]
Woot!

**Pull request merged**